packages/nestjs-retry/package.json

```diff
@@ -60,8 +60,8 @@
     "@geersch/retry": "workspace:^"
   },
   "peerDependencies": {
-    "@nestjs/common": "^9.0.0 || ^10.0.4",
-    "@nestjs/core": "^9.0.0 || ^10.0.4",
-    "reflect-metadata": "^0.1.12"
+    "@nestjs/common": "^10.0.0",
+    "@nestjs/core": "^10.0.0",
+    "reflect-metadata": "^0.2.1"
```

Comment:
Could be 

```
"reflect-metadata": "^0.1.12 || ^0.2.0"
```

To allow for >= `0.1.12` or >= `0.2.0`. With a `0.y.z` release every minor version bump is considered breaking.